modules/accredited_representative_portal/app/services/poa_request_search_filter_service.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+DEFAULT_PARAMS = {
+  status: 'Pending',
+  sortField: 'ar_power_of_attorney_requests.created_at',
+  sortDirection: 'desc',
+  pageNumber: 0,
+  pageSize: 10
+}.freeze
+
+class PoaRequestSearchFilterService
+  def self.handle_filter(filter_params)
+    result = PoaRequestQueryParamsSchema.call(DEFAULT_PARAMS.merge(filter_params))
```

Comment:
Turn this class into a service object and place the normalization inside #initialize